src/Executor/Runner/Docker.php

```diff
@@ -0,0 +1,1131 @@
+<?php
+
+namespace OpenRuntimes\Executor\Runner;
+
+use Appwrite\Runtimes\Runtimes;
+use Exception;
+use OpenRuntimes\Executor\Usage;
+use OpenRuntimes\Executor\Validator\TCP;
+use Swoole\Process;
+use Swoole\Table;
+use Swoole\Timer;
+use Throwable;
+use Utopia\CLI\Console;
+use Utopia\Http\Http;
+use Utopia\Http\Response;
+use Utopia\Logger\Log;
+use Utopia\Orchestration\Orchestration;
+use Utopia\Storage\Device\Local;
+use Utopia\System\System;
+
+use function Swoole\Coroutine\batch;
+
+class Docker extends Adapter
+{
+    public function __construct(private readonly Orchestration $orchestration)
+    {
+    }
+
+    /**
+     * @param Table $activeRuntimes
+     * @param string[] $networks
+     * @param Table $statsHost
+     * @param Table $statsContainers
+     * @return void
+     */
+    public function init(Table $activeRuntimes, array $networks, Table $statsHost, Table $statsContainers): void
```

Comment:
Will we need to handle state like this also in the other adapter? If we do, can we have a parent class that handles this on a higher level? This will help us make adapters leaner and avoid having to pass this knowledge/tasks between adapters. 

Can we also move this logic to the constructor instead of having an init method?